go/vt/sqlparser/sql.y

```diff
@@ -938,10 +978,157 @@ value_expression:
   {
     $$ = $1
   }
+
 | VALUES openb sql_id closeb
   {
     $$ = &ValuesFuncExpr{Name: $3}
   }
+| CONVERT openb expression ',' convert_type closeb
```

Comment:
Ok, does it make sense to move up a level then?  Perhaps these should be in the `expression:` grammar itself? I don't have a huge objection here, just commenting that all of the recursive rules in `value_expression` grammar reference another `value_expression`, while these new ones reference `expression`